var/spack/repos/builtin/packages/gatk/package.py

```diff
@@ -30,10 +31,15 @@ class Gatk(Package):
 
     def install(self, spec, prefix):
         mkdirp(prefix.bin)
-
-        # Install all executable non-script files to prefix bin
-        jar_file = 'GenomeAnalysisTK.jar'
-        install(jar_file, prefix.bin)
+        # The list of files to install varies with release...
+        # ... but skip the spack-{build.env}.out files and gatkdoc directory.
+        files = [x for x in glob.glob("*")
+                 if not re.match("^spack-", x) and not re.match("^gatkdoc", x)]
+        for f in files:
+            if os.path.isdir(f):
+                install_tree(f, join_path(prefix.bin, f))
+            else:
+                install(f, prefix.bin)
```

Comment:
I honestly wonder if it's easier to just `install_tree` the entire directory and then delete the stuff you don't want.

Reply:
I'm indifferent, thoughts on just keeping all files?  The gatkdoc directory is only 5% of the total size.

Reply:
I don't use this package, so I'm also indifferent 😄